enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTableVisualization.jsx

```diff
@@ -20,6 +20,19 @@ import cx from "classnames";
 
 const getColumnName = column => column.remapped_to || column.name;
 
+const propTypes = {
+  series: PropTypes.array,
+  visualizationIsClickable: PropTypes.func,
```

Comment:
Not knowing anything about this prop (judging purely from its name), it seems like it should be boolean, no?

Reply:
I thought so too, but it is used as a function

Reply:
🤷‍♂️